entries/c/coinbase.com.json

```
@@ -1,13 +1,13 @@
 {
   "Coinbase": {
     "domain": "coinbase.com",
-    "url": "https://www.coinbase.com/",
     "tfa": [
       "sms",
       "totp",
       "u2f"
     ],
     "documentation": "https://support.coinbase.com/customer/en/portal/articles/1658338",
```

```suggestion
    "documentation": "https://help.coinbase.com/en/coinbase/getting-started/verify-my-account/how-do-i-set-up-2-factor-authentication",
```
Might as well update the link to where it redirects.